wire-ios/WireCommonComponents/FileMetaDataGenerator/FileMetaDataGenerator.swift

```diff
@@ -42,7 +42,7 @@ public final class FileMetaDataGenerator: FileMetaDataGeneratorProtocol {
 
         let thumbnail: Data?
         do {
-            thumbnail = try await previewGenerator.generatePreviewForFile(at: url)
```

Comment:
The `await` was not necessary here - there was a build warning. This change is unrelated to dropping iOS 15